packages/typespec-vscode/src/utils.ts

```diff
@@ -31,9 +30,9 @@ export async function isDirectory(path: string) {
   }
 }
 
-export async function createTempDir(): Promise<string | undefined> {
+export async function createTempDir(tmpRoot: string): Promise<string | undefined> {
```

Comment:
suggest changing to createTempDir(tmpRoot: string, prefix: string) to keep the utility function for general purpose instead of just for openapi3-preview.

Reply:
updated according to comment